packages/strapi-helper-plugin/lib/src/utils/dateFormats.js

```diff
@@ -1,8 +1,8 @@
-const DATE_FORMATS = {
+const dateFormats = {
```

Comment:
This file should stay in the `strapi-plugin-content-manager` as it easy for the user to modify their date format in the plugin.

Reply:
What you can do though in order to leave the ability for the users to customise their date format and to leave this file in the `helper-plugin` is to create a `dateFormat` file in the `content-manager` as follows:

```
import { dateFormats } from 'strapi-helper-plugins';

const formats = {
   ...dateFormats,
  // Customise the format by uncommenting the one you wan to override it corresponds to the type of your field
  // date: 'dddd, MMMM Do YYYY',
  //  datetime: 'dddd, MMMM Do YYYY HH:mm',
  // time: 'HH:mm A',
  // timestamp: 'dddd, MMMM Do YYYY HH:mm',
};
```